app/containers/SendTips/saga.js

```diff
@@ -173,6 +177,44 @@ export function* sendTipsWorker({ resetForm, val, questionId, answerId }) {
   }
 }
 
+export function* sendTipsNotificationWorker({
+  data,
+  questionId,
+  answerId,
+  transactionId,
+  block,
+}) {
+  try {
+    let attempts = 1;
+    while (attempts <= 5) {
+      // delay before notifications tips service call
+      yield new Promise(res => {
+        setTimeout(() => {
+          res();
+        }, 700);
```

Comment:
Let's increase this timeout to 2 seconds and reduce number of attempts to 3.

Reply:
updated